Contributing-code-to-Oppia.md

```diff
@@ -184,8 +185,7 @@ If an issue hasn't got someone assigned to it, and there's no existing PR for th
 
   ```console
   git blame -L 40,60 file-name
-  ```
-
+  
```

Comment:
Why are you changing this? It looks like you didn't preview the file before submitting since now all the text below is in code font as well.

Please make sure to preview the changes before asking for a review, and make sure the "Files Changed" tab contains exactly the required changes and nothing else. I'm going to stop reviewing here because this PR isn't ready for review.